x-pack/elastic-agent/elastic-agent.reference.yml

```diff
@@ -14,20 +14,19 @@ outputs:
     username: elastic
     password: changeme
 
-datasources:
-  - namespace: default
+inputs:
+  - type: system/metrics
+    dataset.namespace: default
```

Comment:
Do you already have support for `dataset.type`? I don't think we need it in this PR but curious if you added it?

Reply:
yep dataset.type is supported